src/modules/options/components/StreamingServiceOptions.tsx

```diff
@@ -1,23 +1,60 @@
-import { List } from '@material-ui/core';
+import { Grid, ListItem, Tooltip, Typography } from '@material-ui/core';
+import HelpIcon from '@material-ui/icons/Help';
 import * as React from 'react';
-import { StreamingServiceOption } from './StreamingServiceOption';
+import { StreamingServiceValue } from '../../../common/BrowserStorage';
+import { I18N } from '../../../common/I18N';
 import { StreamingServiceId } from '../../../streaming-services/streaming-services';
+import { StreamingServiceOption } from './StreamingServiceOption';
 
 interface StreamingServiceOptionsProps {
-	options: Record<StreamingServiceId, boolean>;
+	options: Record<StreamingServiceId, StreamingServiceValue>;
 }
 
 export const StreamingServiceOptions: React.FC<StreamingServiceOptionsProps> = (
 	props: StreamingServiceOptionsProps
 ) => {
 	const { options } = props;
 	return (
-		<List classes={{ root: 'options--streaming-service' }}>
-			{(Object.entries(options) as [StreamingServiceId, boolean][])
-				.sort(([idA], [idB]) => idA.localeCompare(idB))
-				.map(([id, value]) => (
-					<StreamingServiceOption key={id} id={id} value={value} />
-				))}
-		</List>
+		<ListItem>
+			<Grid container>
+				<Grid container className="options-grid-container">
+					<Grid item className="options-grid-item options-grid-item--centered" xs={4}>
```

Comment:
I don't think it looks to good when this is centered.

Reply:
Just the service one?

Reply:
Yeah. It seems to stand out as it has some distance to the text below.